### DISCUSSION TITLE
dynamic parameters of loss functions based on epoch or step

### DISCUSSION THREAD

**Discussion by Minyus** [open-mmlab/mmclassification#1830] [Q&A]
Learning rate can be dynamically modified by a scheduler based on epoch or step number, but can parameters of loss functions (such as gamma and loss_weight of focal loss) be dynamically modified?
Is there a recommended way to read epoch or step number from forward method of custom loss functions?
